content/working_with/console/widgets/deploymentsView.md

```diff
@@ -283,6 +283,9 @@ The _Run Workflow_ bulk action will show a modal that allows selecting a
 workflow that will be executed on each deployment that matches the current
 filter.
 
+After selecting workflow, fields allowing to modify values of the workflow parameters will appear.   
+Fields will be initialized with default values of the workflow parameters.
```

Comment:
What do you think about rephrasing it a bit:
```suggestion
After selecting a workflow, user will be able to adjust workflow parameters. Only simple type parameters (<list of supported types goes here>) which are common for deployments matching the current filter are presented. Parameters with default values will have the initial value set.
```
?

Reply:
Sure, seems like a nice adjustment 🎉

Reply:
Adjusted within 6963aaf51124c3bd03f773a8eb4e1a9bbb1c31a6